src/components/AddBookForm.js

```diff
@@ -25,34 +25,37 @@ const AddBookForm = () => {
   };
 
   return (
-    <div className="new-AddBook">
-      <h2>ADD NEW BOOK</h2>
-      <form className="new-bookform">
-        <input
-          type="text"
-          className="new-bookTitle"
-          placeholder="Book title"
-          name="title"
-          value={formData.title}
-          onChange={handleChange}
-          required
-        />
-        <br />
-        <input
-          type="text"
-          className="new-authorName"
-          placeholder="Author Name"
-          name="author"
-          value={formData.author}
-          onChange={handleChange}
-          required
-        />
-        <br />
-        <button type="submit" onClick={handleSubmit}>
-          ADD BOOK
-        </button>
-      </form>
-    </div>
+    <>
+      <div className="line" />
+      <div className="new-AddBook">
+        <h2>ADD NEW BOOK</h2>
+        <form className="new-bookform">
+          <input
+            type="text"
+            className="new-bookTitle"
+            placeholder="Book title"
+            name="title"
+            value={formData.title}
+            onChange={handleChange}
+            required
+          />
+          <br />
+          <input
+            type="text"
+            className="new-authorName"
+            placeholder="Author Name"
+            name="author"
+            value={formData.author}
+            onChange={handleChange}
+            required
+          />
+          <br />
+          <button type="submit" onClick={handleSubmit}>
+            ADD BOOK
+          </button>
+        </form>
+      </div>
+    </>
   );
 };
 
```

Comment:
- [ ] Styling should be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/) Font size and weight, colors of the headers didn't match that in zeplin. As you can see on the screenshot below only title and category is required here not title and author.

| Zeplin :+1: | Yours |
| --- | --- |
| ![Screenshot from 2023-09-29 06-54-40](https://github.com/mryeminthu/bookstore/assets/63087491/1cb43c37-1af0-48f6-a388-0ec0fabe3c88) | ![Screenshot from 2023-09-29 06-54-59](https://github.com/mryeminthu/bookstore/assets/63087491/ee149d7a-9b54-436d-b626-1942e8dae766) |